BraintreeCore/src/main/java/com/braintreepayments/api/core/LinkType.kt

```diff
@@ -0,0 +1,11 @@
+package com.braintreepayments.api.core
+
+/**
+ * Used to describe the link type for analytics
+ * Note: This enum is exposed for internal Braintree use only. Do not use.
+ * It is not covered by Semantic Versioning and may change or be removed at any time.
+ */
+public enum class LinkType(val stringValue: String) {
```

Comment:
Could we use the `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)` to scope this enum to internal use?